src/main/resources/db/migration/0300/V348__FundingEntities.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE accelerator.funding_entities
+(
+    id   BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE TABLE accelerator.funding_entity_projects
+(
+    funding_entity_id BIGINT NOT NULL REFERENCES accelerator.funding_entities,
+    project_id        BIGINT NOT NULL REFERENCES projects
```

Comment:
Should these have `ON DELETE CASCADE`?

Reply:
Indeed! I meant to add that and then got distracted.